packages/chart/src/FigureChartModel.ts

```diff
@@ -265,6 +265,15 @@ class FigureChartModel extends ChartModel {
         ? this.dh.plot.DownsampleOptions.DISABLE
         : this.dh.plot.DownsampleOptions.DEFAULT
     );
+
+    if (this.figure.errors.length > 0) {
+      // We don't have a toast or anything we show for errors; just log the error for now
+      log.error('Errors in figure', this.figure.errors);
+    }
```

Comment:
Since this is just logged, will the panel be blank? Should we throw something to get caught and displayed by the error boundary?

Reply:
I think _ideally_ we'd display a toast, or some other type of error message; but also it's possible that we can still plot some of the other series that do not have errors, so I don't think we want to just throw and display the error at the error boundary.

Reply:
It now displays a button in the button bar.